.github/workflows/release-patch.yml

```diff
@@ -45,36 +50,42 @@ jobs:
                 majorVersion,
               });
 
-              const goodCommit = await getLatestGreenCommit({
-                github,
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                branch: `release-x.${majorVersion}.x`,
-              });
+              // only allow overriding the commit SHA for workflow_dispatch events with a provided sha
+              const overrideSha = String(context.payload.inputs.override_sha).trim();
+              const isValidOverride = context.eventName === 'workflow_dispatch' && overrideSha.length === 40;
+
+              const releaseCommit = isValidOverride
```

Comment:
if you put in `076063` instead of  `0760630f106ff9796184527ab06ae50f84486b70` it looks like this just ignores it and uses the latest commit. Can we loudly error or something? If we require the full sha we need to let the person know?